drydock_provisioner/drivers/oob/pyghmi_driver/__init__.py

```diff
@@ -99,20 +99,16 @@ def execute_task(self, task_id):
                         task_id=subtask.get_id(), node=n)
             runner.start()
 
-        # Wait for subtasks to complete
-        # TODO need some kind of timeout
-        i = 0
-        while len(incomplete_subtasks) > 0:
+        attempts = 0
+        while len(incomplete_subtasks) > 0 and attempts <= getattr(drydock_provisioner.conf.timeouts, task.action, 5):
```

Comment:
Rather than 5, you could use a default config value instead, i.e. whatever the default drydock_provisioner task timeout is?